### PR TITLE
Continue improve unit cases

### PR DESCRIPTION
**Pull request by EricBlack**
- [x] Cryptography test cases
- Update IKeyStore interface due to IsOpen cannot works for account list open status.
- [x] Consensus.DPoS.SideChain test cases.
- [x] Remove completed or outdated test cases todo comments.

### CODE REVIEW

File: AElf.Core.Tests/AElf.Core.Tests.csproj [hunk 1]
```diff
@@ -11,8 +11,12 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
-      <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
+        <PackageReference Include="coverlet.msbuild" Version="2.5.1" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
+        <PackageReference Include="Shouldly" Version="3.0.2" />
```
Comment (loning) [inline AElf.Core.Tests/AElf.Core.Tests.csproj]: <PackageReference Include="Shouldly" Version="3.0.2" />
        <PackageReference Include="xunit" Version="2.4.1" />
        <PackageReference Include="xunit.runner.console" Version="2.4.1" />
        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />

remove

File: AElf.Cryptography.Tests/AElfKeyStoreTests.cs [hunk 2]
```diff
@@ -96,9 +98,13 @@ public async Task Open_Account_WithTimeout()
...
             var addString = address.GetFormatted();
 
-            //Open account
-            var errResult = await _keyStore.OpenAsync(addString, "123", false);
-            errResult.ShouldBe(AElfKeyStore.Errors.None);
+            //Open account with timeout
+            _keyStore.DefaultTimeoutToClose = TimeSpan.FromMilliseconds(200);
+            _keyStore.OpenAsync(addString, "123").Wait();
+            
+            Thread.Sleep(1000);
```
Comment (loning) [inline AElf.Cryptography.Tests/AElfKeyStoreTests.cs]: do not make unit test wait such long time

File: AElf.WebApp.Application.Net.Tests/AElf.WebApp.Application.Net.Tests.csproj [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
         <TargetFramework>netcoreapp2.2</TargetFramework>
-
         <IsPackable>false</IsPackable>
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="coverlet.msbuild" Version="2.5.1" />
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
-        <PackageReference Include="coverlet.msbuild" Version="2.6.0" />
+        <PackageReference Include="Shouldly" Version="3.0.2" />
```
Comment (loning) [inline AElf.WebApp.Application.Net.Tests/AElf.WebApp.Application.Net.Tests.csproj]: do not add new reference

File: AElf.WebApp.Application.TestBase/AElf.WebApp.Application.TestBase.csproj [hunk 4]
```diff
@@ -2,21 +2,20 @@
...
         <PackageReference Include="Microsoft.AspNetCore.Cors" Version="2.2.0" />
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
         <PackageReference Include="Swashbuckle.AspNetCore" Version="4.0.1" />
         <PackageReference Include="Volo.Abp.AspNetCore.TestBase" Version="0.15.0" />
         <PackageReference Include="Volo.Abp.Autofac" Version="0.15.0" />
         <PackageReference Include="Volo.Abp.Core" Version="0.15.0" />
+        <PackageReference Include="coverlet.msbuild" Version="2.5.1" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
         <PackageReference Include="xunit" Version="2.4.1" />
+        <PackageReference Include="xunit.runner.console" Version="2.4.1" />
```
Comment (loning) [inline AElf.WebApp.Application.TestBase/AElf.WebApp.Application.TestBase.csproj]: remove

File: AElf.Cryptography.Tests/AElfKeyStoreTests.cs (hunk 2 above)
Comment (EricBlack) [reply AElf.Cryptography.Tests/AElfKeyStoreTests.cs]: have changed to 60ms.

File: AElf.WebApp.Application.Net.Tests/AElf.WebApp.Application.Net.Tests.csproj (hunk 3 above)
Comment (EricBlack) [reply AElf.WebApp.Application.Net.Tests/AElf.WebApp.Application.Net.Tests.csproj]: removed those duplicate reference.

File: AElf.WebApp.Application.TestBase/AElf.WebApp.Application.TestBase.csproj (hunk 4 above)
Comment (EricBlack) [reply AElf.WebApp.Application.TestBase/AElf.WebApp.Application.TestBase.csproj]: removed.

**Pull request merged**